include/realizations/catchment/Formulation_Manager.hpp

```diff
@@ -246,6 +246,12 @@ namespace realization {
                     return "";
             }
 
+            void finalize() {
+                for (auto const& fmap: formulations) {
```

Comment:
Need to double-check the relationship between `formulations` and `domain_formulations` (if that even persists in present `master`)

Reply:
@hellkite500 Care to comment? There aren't really descriptive comments in the code

Reply:
After looking at the construction logic, I went ahead with treating them the same.